README.md

```diff
@@ -8,85 +8,75 @@
+## Why Godog/Cucumber
 
-The following about section was taken from
-[cucumber](https://cucumber.io/) homepage.
+### A single source of truth
 
-## Notice:
+Godog merges specification and test documentation into one cohesive whole.
 
-**If your project depend on the master version of godog instead of a specific release, please read this.**
+### Living documentation
 
-Due to dependency changes in a coming merge to master, including breaking changes, you should update how you install or depend on godog so that you have a version specified.
+Because they're automatically tested by Godog, your specifications are
+always bang up-to-date.
 
-## Install
-```
-go get github.com/cucumber/godog/cmd/godog@v0.9.0
-```
-Adding `@v0.9.0` will install v0.9.0 specifically instead of master.
+### Focus on the customer
 
-Running `within the $GOPATH`, you would also need to set `GO111MODULE=on`, like this:
-```
-GO111MODULE=on go get github.com/cucumber/godog/cmd/godog@v0.9.0
-```
+Business and IT don't always understand each other. Godog's executable specifications encourage closer collaboration, helping teams keep the business goal in mind at all times.
 
-## About
+### Less rework
 
-#### A single source of truth
+When automated testing is this much fun, teams can easily protect themselves from costly regressions.
 
-Cucumber merges specification and test documentation into one cohesive whole.
+### Read more
+- [Behaviour-Driven Development](https://cucumber.io/docs/bdd/)
+- [Gherkin Reference](https://cucumber.io/docs/gherkin/reference/)
 
-#### Living documentation
+## Contributions
 
-Because they're automatically tested by Cucumber, your specifications are
-always bang up-to-date.
+Godog is a community driven Open Source Project within the Cucumber organization, it is maintained by a handfull of developers, but we appreciate contributions from everyone.
+
+If you are interested in developing Godog, we suggest you to visit on of our slack channels.
 
-#### Focus on the customer
+Feel free to open a pull request. Note, if you wish to contribute larger changes or an extension to the exported methods or types, please open an issue before and visit us in slack to discuss the changes.
+
+Reach out to the community on our [Cucumber Slack Community](https://cucumberbdd.slack.com/).
+Join [here](https://cucumberbdd-slack-invite.herokuapp.com/).
 
-Business and IT don't always understand each other. Cucumber's executable
-specifications encourage closer collaboration, helping teams keep the
-business goal in mind at all times.
+### Popular Cucumber Slack channels for Godog:
+- [#help-godog](https://cucumberbdd.slack.com/archives/CTNL1JCVA) - General Godog Adoption Help
+- [#committers-go](https://cucumberbdd.slack.com/archives/CA5NJPDJ4) - Golang focused Cucumber Contributors
+- [#committers](https://cucumberbdd.slack.com/archives/C62D0FK0E) - General Cucumber Contributors
 
-#### Less rework
+## Install
```

Comment:
I feel like the install step should be higher up, but lets see how things go here.